package com.lzm.javase.exception2;

public class ArithmeticExceptionDemo {
	
	public void runExceptino() {
		System.out.println(1/0);
	}

}
